pyrene/src/components/Filter/FilterComponents/FilterBar.tsx

```diff
@@ -28,9 +63,15 @@ import FilterTag from './FilterTag';
  *          |- MultiSelect: type of Filter input, expects [{value:, label: }, {valueX:, labelX: }...]
  */
 
-export default class FilterBar extends React.Component {
+export default class FilterBar extends React.Component<FilterBarProps, FilterBarState> {
+
+  static displayName = 'FilterBar';
 
-  constructor(props) {
+  static defaultProps = {
+    onFilterSubmit: () => null,
```

Comment:
I prefer undefined over empty function calls, empty function calls are still called.
And I personally find it more readable to know that a function does not exist.